models/public/mozilla-deepspeech-0.8.2/pbmm_to_pb.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+#
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import argparse
+
+try:
+    from tensorflow import GraphDef
+except ImportError:
+    from tensorflow.compat.v1 import GraphDef
+
+from mds_convert_utils.memmapped_file_system_pb2 import MemmappedFileSystemDirectory
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Convert memory-mapped Tensorflow protobuf format into plain Tensorflow protobuf (GraphDef)")
+    parser.add_argument('input', type=str, help="Input .pbmm filename")
+    parser.add_argument('output', type=str, help="Output .pb filename")
+    parser.add_argument('--overwrite', action='store_true', help="Overwrite output file if exists")
```

Comment:
This is an unnecessary option; it's always set when the script is used.

Reply:
Done. Removed --overwrite and --silent options in both scripts. This was done for "failsafety", to not allow the user accidentally overwrite his own files. LM files are huge, it would be a pain to download them again.